SMLHelper/Patchers/CraftDataPatcher_Subnautica.cs

```diff
@@ -58,148 +59,73 @@ private static void PatchForSubnautica(HarmonyInstance harmony)
             harmony.Patch(AccessTools.Method(typeof(CraftData), nameof(CraftData.IsBuildableTech)),
                prefix: new HarmonyMethod(AccessTools.Method(typeof(CraftDataPatcher), nameof(GetBuildablePrefix))));
 
+            harmony.Patch(AccessTools.Method(typeof(CraftData), nameof(CraftData.GetUseEatSound)),
```

Comment:
Btw, if you're editing this file, you can also try to use that `PatchUtils.PatchClass` stuff here

Reply:
Would you be good to checkout this branch and make that update?

Reply:
That's quite a lot of clean up.